lib/iron_bank/resources/invoice.rb

```diff
@@ -8,7 +8,11 @@ module Resources
     class Invoice < Resource
       # See the comment for the instance method `#body`
       def self.excluded_fields
-        super + %w[Body]
+        super + separate_query_fields
+      end
+
+      def self.separate_query_fields
```

Comment:
WDYTA moving to a private constant (eg., `SINGLE_RESOURCE_QUERY_FIELDS`)?

Reply:
Good point for the naming, but won't be using a constant in just that file since it would require introspection to check if the class defines it.